privacyidea/lib/resolvers/SCIMIdResolver.py

```diff
@@ -133,7 +133,7 @@ def getUserId(self, loginName):
         #                                          ("userName", loginName)})
         #return res.get("Resources", [{}])[0].get("externalId")
         # It seems that the userName is the userId
-        return loginName
+        return str(loginName)
```

Comment:
I haven't tested it, but this may also break under Python 2.7 -- we should use ``text_type`` instead.

Reply:
Fixed in 61743ea